misc/ballerina-bindgen/src/test/resources/mvn-test-resources/balProject/tests/main_test.bal

```diff
@@ -0,0 +1,7 @@
+import ballerina/io;
```

Comment:
Add copyright header if we usually add them to test files.

Reply:
Most of the test `.bal` files do not seem to contain the copyright header.